pages/common/nohup.md

```diff
@@ -4,4 +4,4 @@
 
 - Run process that can live beyond the terminal:
 
-`nohup {{command options}}`
+`nohup {{command_with_options}}`
```

Comment:
Yeah, is definitively better. What about this is option?
```suggestion
`nohup {{command}} {{command_options}}`
```

Reply:
Either would work.

Reply:
I found in the codebase three `{{command}} {{command_arguments}}` , one `{{command}} {{args}}`, and a couple `{{args}}`. Just to add some more doubts :)